WooCommerce/WooCommerceTests/ViewModels/CardPresentPayments/CollectOrderPaymentUseCaseTests.swift

```diff
@@ -154,11 +154,10 @@ final class CollectOrderPaymentUseCaseTests: XCTestCase {
         }
 
         // Then
-        XCTAssertNotNil(result?.failure as? PaymentCaptureOrchestrator.NotValidAmountError)
+        XCTAssertNotNil(result?.failure as? CollectOrderPaymentUseCase.NotValidAmountError)
 
         let indexOfEvent = try XCTUnwrap(analyticsProvider.receivedEvents.firstIndex(where: { $0 == "card_present_collect_payment_failed"}))
         let eventProperties = try XCTUnwrap(analyticsProvider.receivedProperties[indexOfEvent])
-        XCTAssertEqual(eventProperties["card_reader_model"] as? String, Mocks.cardReaderModel)
```

Comment:
Since we do not connect to the card reader, we do not have this information available to track